beacon-chain/db/state.go

```diff
@@ -156,8 +156,9 @@ func (db *BeaconDB) SaveJustifiedState(beaconState *pb.BeaconState) error {
 
 // SaveFinalizedState saves the last finalized state in the db.
 func (db *BeaconDB) SaveFinalizedState(beaconState *pb.BeaconState) error {
+	finalizedSlot := beaconState.FinalizedEpoch * params.BeaconConfig().SlotsPerEpoch
```

Comment:
we can use `helper.SlotToEpoch(beaconState.FinalizedEpoch)` here

Reply:
I didn't wan't to bring the core package into here, imo the db package should be as self-contained as possible

Reply:
I think we should have those helpers in the shared package, we also need them in the validator client btw

Reply:
It can be a separate PR

Reply:
ag